util/find-doc-nits

```diff
@@ -884,6 +894,19 @@ if ( $opt_c ) {
 
 # Preparation for some options, populate %name_map and %link_map
 if ( $opt_l || $opt_u || $opt_v ) {
+    if ( $SRCDIR ne $BLDDIR ) {
+        # If building out-of-tree, foo.pod.in in $SRCDIR should be found as
+        # foo.pod in $BLDDIR;
+        foreach my $in ( glob('doc/*/*.pod.in doc/internal/*/*.pod.in') ) {
```

Comment:
This can't possibly work in an out-of-source build tree

Reply:
yes, it needs $SRCDIR prepended.  will fix.